userdata/testAPI.py

```diff
@@ -0,0 +1,18 @@
+'''
+Created on Mar 16, 2015
+
+@author: raju332
+'''
+import VeloAPI

```

Comment:
Its not getting run. It was given to us just as an example of how to use the API, but at this point I dont think we need it anymore. The functionality that it uses is getting tested in the actual django tests.